packages/alert-message-react/src/AlertMessage.tsx

```diff
@@ -7,6 +7,10 @@ import { MessageIcon } from "./common/MessageIcon";
 
 type messageTypes = "info" | "error" | "success" | "warning";
 
+/**
+ * @deprecated Denne komponenten blir omdøpt til SystemMessage
+ */
+
```

Comment:
Denne må stå rett over eksportene av de forskjellige variantene for å dukke opp i Intellisense